elitzur-core/src/test/scala/com/spotify/elitzur/validators/ValidatorTest.scala

```diff
@@ -0,0 +1,207 @@
+package com.spotify.elitzur.validators
```

Comment:
I think some of our core tests got dropped when we open sourced. Adding some simple testing here so we aren't relying on ValidatorDoFnTest as much